gocourse17/service/review.go

```diff
@@ -0,0 +1,42 @@
+package service
+
+const (
+	poor int = iota + 1
+	fair
+	good
+	great
+	excellent
+)
+
+type Review struct {
+	CargoState       int
+	ServiceQuality   int
+	FulfillmentSpeed int
+}
```

Comment:
В нас обмежений набір значень рейтингу, варто використовувати константи замість `int`:
```suggestion
type Rating int

const (
	RatingPoor Rating = iota + 1
	RatingFair
	RatingGood
	RatingGreat
	RatingExcellent
)

type Review struct {
	CargoState       Rating
	ServiceQuality   Rating
	FulfillmentSpeed Rating
}
```